src/rtp.c

```diff
@@ -201,13 +201,6 @@ rtp_check_packet(packet_t *packet)
         if (!(stream_is_complete(stream))) {
             stream_complete(stream, src);
             stream_set_format(stream, format);
-            // Check if an stream in the opposite direction exists
-            if (!(reverse = rtp_find_call_stream(stream->media->msg->call, stream->dst, stream->src))) {
-                reverse = stream_create(stream->media, stream->src, PACKET_RTP);
-                stream_complete(reverse, stream->dst);
-                stream_set_format(reverse, format);
-                call_add_stream(msg_get_call(stream->media->msg), reverse);
-            }
```

Comment:
The images below show that the _reverse_  stream is required. In the screens, the 200 OK contains the address 192.168.175.110:8262, but not packet is seen with that destination.

The UAC has been waiting until a packet have been received in the configured address in its INVITE (5.196.32.131:55772) and starts sending data to that source address (212.55.31.125:8262) totally ignoring the address configured in the 200 OK. This _sending_ stream packets are detected as the _reverse stream_ of the configured in the INVITE. That's the reason why in the second screen only three streams are printed.